modules/resetdigitalBidAdapter.js

```diff
@@ -0,0 +1,144 @@
+
+import * as utils from '../src/utils.js';
+import { config } from '../src/config.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+const BIDDER_CODE = 'resetdigital';
+
+function getDomain () {
+  if (!utils.inIframe()) {
+    return window.location.hostname
+  }
+  let origins = window.document.location.ancestorOrigins
+  if (origins && origins.length > 0) {
+    return origins[origins.length - 1]
+  }
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [ 'banner', 'video' ],
+  isBidRequestValid: function(bid) {
+    return (bid.params.pubId !== null || bid.params.zoneId !== null);
+  },
+  buildRequests: function(validBidRequests, bidderRequest) {
+    let stack = (bidderRequest.refererInfo &&
+      bidderRequest.refererInfo.stack ? bidderRequest.refererInfo
```

Comment:
I think the value if the condition proves true was meant to be bidderRequest.refererInfo.stack, no?